csunplugged/templates/pluggingitin/index.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base-students.html" %}
+
+{% load i18n %}
+{% load static %}
+{% load django_bootstrap_breadcrumbs %}
+
+{% block title %}
+  {% trans "Plugging It In" %}
+{% endblock title %}
+
+{% block breadcrumbs %}
+  {% breadcrumb "Plugging it in" "pluggingitin:index" %}
+{% endblock breadcrumbs %}
+
+{% block page_heading %}
+  <h1>{% trans "Plugging It In" %}</h1>
+{% endblock page_heading %}
+
+{% block content %}
+  {% if programming_topics %}
+    <p>{% trans "[PLUGGING IT IN BLURB.....] Lorem ipsum dolor sit amet, nec metus leo pulvinar urna. Vestibulum ut, wisi neque rutrum. Euismod volutpat integer libero justo, sit erat dapibus tortor duis arcu elit, class nisl massa dictum eleifend adipiscing, a nec ridiculus risus egestas augue id. Ligula non nulla at curabitur feugiat, vestibulum sit at vestibulum nam arcu." %}</p>
+    <div class="row justify-content-between">
+      {% for topic in programming_topics %}
```

Comment:
For this iteration of the testing, it would be good to have the six developed questions at the top of this page. Possibly with the topics below or hidden altogether.